pywren/wrenhandler.py

```diff
@@ -60,6 +61,8 @@ def download_runtime_if_necessary(s3_client, runtime_s3_bucket, runtime_s3_key):
 
     """
 
+    lock = open("/tmp/runtime_download_lock", "a")
+    fcntl.lockf(lock, fcntl.LOCK_EX)
     # get runtime etag
     runtime_meta = s3_client.head_object(Bucket=runtime_s3_bucket,
```

Comment:
I agree that supporting different runtimes can be out of scope for now. But can we detect that a user is trying to do that and fail here ? i.e. if you try to download a new runtime while another handler is using an existing runtime or something like that. Or is there a simpler way to detect this by looking at the job definitions ?

Reply:
I think theres a discussion about this on slack, I think we can support multiple runtimes pretty easily...